src/main/java/net/blay09/mods/eiramoticons/addon/pack/TwitchSubscriberPack.java

```diff
@@ -21,43 +22,36 @@
 
 public class TwitchSubscriberPack extends AbstractEmotePack {
 
-	private String template;
-
 	public TwitchSubscriberPack(String regexFilter) {
 		try {
 			Pattern pattern = Pattern.compile(regexFilter);
 			Matcher matcher = pattern.matcher("");
 			Reader reader = TwitchEmotesAPI.newSubscriberEmotesReader(false);
 			Gson gson = new Gson();
-			JsonObject root;
+			JsonObject emoteList;
 			try {
-				root = gson.fromJson(reader, JsonObject.class);
+				emoteList = gson.fromJson(reader, JsonObject.class);
 			} catch (Exception e) {
 				reader = TwitchEmotesAPI.newSubscriberEmotesReader(true);
 				try {
-					root = gson.fromJson(reader, JsonObject.class);
+					emoteList = gson.fromJson(reader, JsonObject.class);
 				} catch (Exception e2) {
 					throw new EmoteLoaderException(e2);
 				}
 			}
-			if(root != null) {
-				template = getJsonString(getJsonObject(root, "template"), "small");
-				JsonObject channels = getJsonObject(root, "channels");
-				for (Map.Entry<String, JsonElement> channelEntry : channels.entrySet()) {
-					if (channelEntry.getKey().equals("turbo")) {
-						continue;
-					}
+			if (emoteList != null) {
+				for (Map.Entry<String, JsonElement> channelEntry : emoteList.entrySet()) {
 					JsonObject channel = channelEntry.getValue().getAsJsonObject();
-					String title = getJsonString(channel, "title");
-					JsonArray emotes = getJsonArray(channel, "emotes");
-					for (int i = 0; i < emotes.size(); i++) {
-						JsonObject emote = emotes.get(i).getAsJsonObject();
+					String channelName = getJsonString(channel, "channel_name");
```

Comment:
This should be `display_name`, which is the properly cased version of the channel name (e.g. zeekdageek vs ZeekDaGeek)